autotelic/fastify-openapi-autoload
# feat: security handlers

# Summary

- Closes https://github.com/autotelic/shakti-v2/issues/4
- Add jwt/jwks security handlers
- Update example to run on https and use new security handlers feature

# Test Plan

- Run tests
- Run through the README to get the example app running
- Send a request to `/foo` with a bearer token to confirm the route responds with `{ userId: 123 }`
- Try without a bearer token (or change any part of the security chain) to confirm the route fails permissions

TODO:
- update default security handler (`makeSecurityHandler`) with reasonable defaults for it's consumers
